mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -309,6 +318,31 @@ func GenerateDefsForCoreRegistrant(model utils.ModelCSV) error {
 	return nil
 }
 
+func parseModelSheet(url string) (*utils.ModelCSVHelper, error) {
+	modelCSVHelper, err := utils.NewModelCSVHelper(url, "Models", sheetGID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = modelCSVHelper.ParseModelsSheet(false)
+	if err != nil {
+		return nil, ErrGenerateModel(err, "unable to start model generation")
+	}
+	return modelCSVHelper, nil
+}
+
+func parseComponentSheet(url string) (*utils.ComponentCSVHelper, error) {
+	compCSVHelper, err := utils.NewComponentCSVHelper(url, "Components", componentSpredsheetGID)
+	if err != nil {
+		return nil, err
```

Comment:
Missing specific error message.

Reply:
The specifics of the error is being returned from the function NewComponentCSVHelper